treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -20,7 +20,6 @@
     keywords: perf-alert, regression
     default_component: Performance
     default_product: Testing
-    cc_list: rstewart@mozilla.com
```

Comment:
I know there's another team looking over `build_metrics`. We should rather look for another representative' s email.

Reply:
Triage owner will anyway be cc'ed. The other tempalte have no default cc_list al wel.